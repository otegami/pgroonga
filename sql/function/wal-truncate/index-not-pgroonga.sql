CREATE TABLE memos (
  id integer
);

CREATE INDEX index ON memos (id);

SELECT pgroonga_wal_truncate('index');

DROP TABLE memos;
