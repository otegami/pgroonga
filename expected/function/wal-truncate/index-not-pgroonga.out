CREATE TABLE memos (
  id integer
);
CREATE INDEX index ON memos (id);
SELECT pgroonga_wal_truncate('index');
ERROR:  pgroonga: [wal][truncate][index] not PGroonga index: <index>
DROP TABLE memos;
